packages/cmake/Modules-Platform-Android-Determine-Compiler.cmake.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/Modules/Platform/Android/Determine-Compiler.cmake b/Modules/Platform/Android/Determine-Compiler.cmake
+index a03ebcc29..34d527741 100644
+--- a/Modules/Platform/Android/Determine-Compiler.cmake
++++ b/Modules/Platform/Android/Determine-Compiler.cmake
+@@ -31,6 +31,14 @@ elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
+   set(_ANDROID_HOST_EXT "")
+ elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
+   set(_ANDROID_HOST_EXT ".exe")
++elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Android")
++  macro(__android_determine_compiler lang)
++  # Do nothing
++  endmacro()
++  if(NOT CMAKE_CXX_COMPILER_NAMES)
++    set(CMAKE_CXX_COMPILER_NAMES c++)
```

Comment:
Just added this [as Linux does too](https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/Platform/Linux-Determine-CXX.cmake).